pkgs/by-name/rs/rspamd/package.nix

```diff
@@ -80,6 +82,8 @@ stdenv.mkDerivation rec {
     "-DDBDIR=/var/lib/rspamd"
     "-DLOGDIR=/var/log/rspamd"
     "-DLOCAL_CONFDIR=/etc/rspamd"
+    "-DENABLE_BLAS=${if withBlas then "ON" else "OFF"}"
```

Comment:
Use `lib.cmakeBool`?

Reply:
That wouldn't work as they explicitly check for the string "ON". Probably a bug in their cmake but I didn't bother to fix it.